activerecord/lib/active_record/database_configurations/url_config.rb

```diff
@@ -57,6 +57,17 @@ def initialize(env_name, name, url, configuration_hash = {})
         @configuration_hash.freeze
       end
 
+      def query_cache # :nodoc:
+        case value = configuration_hash[:query_cache]
```

Comment:
I think we might want to do this in `initialize` so that we don't redo it on every access.

Reply:
Ok, I didn't realize that was already happening for `"false"`. Added 15606ec2ff.

Reply:
Whoops fixed tests in af9b0f2be4